Resources/converter.swift

```diff
@@ -453,15 +461,15 @@ let exactStartMeasure = startMeasures[0].measure
 let rawEnd = rawOperation
   .dropFirst(startMeasures[0].matchedChars)
   .trimmingCharacters(in: .whitespacesAndNewlines)
-  .removingPrefixes(["to ", "as ", "in "])  // Remove connection words  
+  .removingPrefixes(["to ", "as ", "in "])  // Remove connection words
 
 // When only one starting measure matches, convert
 let endMeasures = {
   // Measures which make sense to convert to
-  let suitableEnds = allMeasures.filter {
-    type(of: exactStartMeasure.unit) == type(of: $0.unit)  // Same unit type, so we can convert
-      && exactStartMeasure.unit != $0.unit  // Remove starting measure
-  }
+    let suitableEnds = allMeasures.filter {
+        return type(of: exactStartMeasure.unit).baseUnit() == type(of: $0.unit).baseUnit()  // Same unit type, so we can convert
```

Comment:
Please don’t add unnecessary code (the `return`).

Reply:
Can’t you make the change without requiring the extra call to `baseUnit()`? I’d rather not have this happen every time for every unit because of a single one.